### ISSUE TITLE
Doors sounds only work one direction

### ISSUE THREAD

**Issue by TelloViz** [OpMonTeam/OpMon-Archive#127]
Doors activate sound upon entering a building but not upon leaving.

I'm new around here so let me know if I submitted this wrong :)

**Comment by milkyroute** [comment]
In fact, it's only something the game doesn't do because we haven't programmed it. This is quite minor yet, so I won't fix this soon, but I keep it because it has to be done.

**Comment by TelloViz** [comment]
So I found the door event code where it is given an entrance sound on creation. Would there simply need to be an exit sound? Or does entering a house count as a different map, therefore all doors in the house would also just have entrance sounds and no additional exit sound code would be needed ? Does this make sense?

**Comment by milkyroute** [comment]
We've planned to do an more complex animation for entering and living houses. Right now, there's no animation at all, the event is not even counted as a door. What the event should do is to teleport the player on the opened door on the outdoor map and then making the character move forward by one tile while the door is closing behind. So, for this, a new event type would have to be created. As you submitted on the Discord server, splitting more the events would be interesting to do this, for the audio for example.

**Comment by TelloViz** [comment]
I think this issue is contingent on #131 so I just want to reference it here.
